appsec/go.mod

```diff
@@ -0,0 +1,9 @@
+module github.com/DataDog/dd-trace-go/appsec
+
+go 1.16
+
+require (
+	github.com/google/uuid v1.3.0
+	github.com/sqreen/go-libsqreen v0.7.2
```

Comment:
https://github.com/DataDog/libddwaf was open-sourced yesterday and I am currently working on porting go-libsqreen to it. I want to separate the bindings update from this PR, and I am aware about the macos compilation issue I introduced here. This is why the target branch of the PR is `appsec` instead of `v1` while this gets fixed by the coming WAF PR.

And yes, the library comes pre-compiled from libddwaf releases, which is a C++ library with a C API we use though CGO.

Reply:
I'm also curious about `libunwind_linux_amd64.a` and `libc++abi_linux_amd64.a`.

Reply:
This is not to use the GNU libstdlibc++ which depends on the glibc and so on, which therefore is glibc-based distributions only. So I had to find a solution for our dear Alpine users by only depending on the standard C library, and to do so, we use the libc++ :-)

You can read more in the Dockerfile I created for that:
https://github.com/DataDog/libddwaf/blob/master/docker/libddwaf/gcc/linux/Dockerfile#L1-L23

And we can discuss more about those things in the new CGO bindings PR coming later this week ^^

Reply:
The macos build issue is now fixed @felixge